WordPress/src/main/res/values/attrs.xml

```diff
@@ -107,4 +107,17 @@
         <attr name="wpTextDialogSubtitle" format="string" />
     </declare-styleable>
 
+    <!--
+        WPLoginInputRow
+    -->
+    <declare-styleable name="wpLoginInputRow">
+        <attr name="wpIconDrawable" format="reference"/>
+        <attr name="wpIconContentDescription" format="string"/>
+        <attr name="android:inputType" />
```

Comment:
Ahh, I didn't realize `passwordToggleEnabled` and `passwordToggleTint` were inherited from `TextInputLayout`.